socs/Lakeshore/README_Lakeshore370.txt

```diff
@@ -0,0 +1,64 @@
+socket functionality replaced with serial, 370 does not support ethernet communication
```

Comment:
Again, important developer notes when framed as this module being adapted from the existing 372 driver code. Could also be incorporated into the docs page for the 370.